src/en/controllers-gui.md

```diff
@@ -17,7 +17,7 @@ To view the URL and login credentials for the GUI, use the following command:
 juju gui
 ```
 
-This will produce output similar to the following:
+This should produce output similar to the following:
```

Comment:
As this is simply a messaging issue on the rare event bootstrap fails to download a GUI. I feel that this should remain as `will` and the issue be resolved in Juju.